src/methods/SANA.cpp

```diff
@@ -1835,20 +1835,52 @@ uint SANA::getHighestIndex() const {
     return highestIndex;
 }
 
-#define LOG10_LOW_TEMP -10.0
-#define LOG10_HIGH_TEMP 10.0
-#define LOG10_NUM_STEPS 20 // looping through 0 <= i <= LOG10_NUM_STEPS
+#define HIGH_TEMP_LIMIT 0.99999
```

Comment:
I know what you mean but these are actually pBad limits not temperature limits. Please rename appropriately.

Reply:
Fixed!